OracleDatabase/SingleInstance/README.md

```diff
@@ -239,17 +239,17 @@ To configure  wallet password, please use the following command:
         # ORACLE_SID default value is ORCLCDB
         docker cp <container name>:/opt/oracle/oradata/clientWallet/<ORACLE_SID> <destination directory>
 
-* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files. You should edit the `HOST` and `PORT` fields accordingly in the `tnsnames.ora` before connecting using TCPS.
-* After `tnsnames.ora` is modified, go inside the downloaded client wallet directory and set TNS_ADMIN for SQL\*Plus by using the `export TNS_ADMIN=$(pwd)` command. Then users can connect via TCPS with, for example, the following commands:
+* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files.
+* To connect to the database via TCPS use EZCONNECT (easy connect naming method) as shown:
```

Comment:
Use the formal anme 'Easy Connect', not the old, mostly internal name EZCONNECT.  Also users are technically not restricted to using Easy Connect.  And there's arguably no need to mention it at all??  So the sentence can be like:
```
To connect to the database via TCPS you can use SQL*Plus like:
```